src/shared/actions/transfers.js

```diff
@@ -259,8 +260,19 @@ export const promoteTransaction = (bundleHash, accountName, powFn) => (dispatch,
                     consistentTail,
                     accountState.transfers[bundleHash].tailTransactions,
                     true,
-                    // If proof of work configuration is set to remote, pass proof of work function as null
-                    remotePoW ? null : powFn,
+                    // If proof of work configuration is set to remote,
+                    // Extend seedStore object with offloadPow
+                    // This property will lead to perform network bound proof-of-work
```

Comment:
```suggestion
                    // This property will lead to perform remote proof-of-work
```

This could be confused with CfB's idea of network-bound PoW:
![image](https://user-images.githubusercontent.com/19519564/50492953-8500de00-09e8-11e9-91e1-ed7742a76a16.png)

Reply:
@rajivshah3 Fixed in https://github.com/iotaledger/trinity-wallet/pull/775/commits/aa685de00950428a514623fb6d7abab48ff21b34